lib/instabug_flutter.dart

```diff
@@ -56,10 +72,9 @@ class InstabugFlutter {
     invocationEvents.forEach((e) {
       invocationEventsStrings.add(e.toString());
     });
-    Map<String, Object> params = {
-      'token': token,
-      'invocationEvents': invocationEventsStrings
-    };
+    final List<dynamic> params = new List<dynamic>();
+    params.add(token);
+    params.add(invocationEventsStrings);
```

Comment:
@alyezz can we change all lists to follow dart best practices. https://dart-lang.github.io/linter/lints/prefer_collection_literals.html
So this part for example will look like this:
`final List<String> params = [token, invocationEventStrings];`